stream_alert_cli/runner.py

```diff
@@ -150,6 +150,10 @@ def add_to_athena_schema(schema, root_key=''):
                     athena_schema[root_key] = {}
 
                 for key_name, key_type in schema.iteritems():
+                    # When using special characters in the beginning or end
+                    # of a column name, they have to be wrapped in backticks
+                    key_name = '`{}`'.format(key_name)
```

Comment:
I am concerned to use backticks here. The `key_name` is getting from the schema which is defined in the log file. I don't think it is safe to use backticks here.

Reply:
@chunyong-lin see the comment above the line which explains why